hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/format/FlinkRowDataReaderContext.java

```diff
@@ -195,7 +195,9 @@ public ClosableIterator<RowData> mergeBootstrapReaders(
       ClosableIterator<RowData> skeletonFileIterator,
       Schema skeletonRequiredSchema,
       ClosableIterator<RowData> dataFileIterator,
-      Schema dataRequiredSchema) {
+      Schema dataRequiredSchema,
+      Option<String[]> partitionFields,
```

Comment:
Should the iterator be fixed too to consider the partition values?

Reply:
Yes, I will need some help getting the tests setup for flink as well